routes/pagination.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function getPaginationParameters(query) {
+  let start = 0
+  let end = 9
+  if (query.start && query.end) {
+    const parsedStart = parseInt(query.start)
+    const parsedEnd = parseInt(query.end)
+    if (parsedStart !== NaN) {
```

Comment:
I know this can have some quirks with strict equality, like `NaN !== NaN == true` 😅 
maybe changing to `Number.isNaN(parsedStart)` is a safer way to handle the failed conversion

Reply:
Nice catch! Fixed using `Number.isNaN()`.